new/LIT7044PrayerSaints.xml

```diff
@@ -0,0 +1,81 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7044PrayerSaints" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Prayer in the Name of all Saints</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="AY"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>The work is a prayer in the name of foreign and Ethiopian saints and martyrs. 
+                    This work also includes a list of Ethiopian place names associated with Ethiopian saints. 
+                    However, as the beginning and ending of the work are missing, the work lacks a clear title.
+                    </p>
+                 </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Prayers"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="AY" when="2024-08-22">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>  
+            <div type="edition">
+                <note>The following incipit and explicit are taken from <ref type="mss" corresp="MD003"/>. 
+                    The beginning and ending of <title type="incomplete" ref="LIT7044PrayerSaints"/> is missing.</note>
```

Comment:
(1) you should not refer from 7044 to 7044, `title ref `is only for linking to other entities

(2) if the beginning and the end are missing then the passages are NOT `incipit `and `explicit`, as they are missing according to what you say. these are just some passages. So reformulate
"The following passages are taken...
The beginning and ending of the text are missing so the incipit/explicit cannot be provided"